mod_reforged/hooks/skills/racial/vampire_racial.nut

```diff
@@ -0,0 +1,31 @@
+::mods_hookExactClass("skills/racial/vampire_racial", function(o) {
+	local create = o.create;
+	o.create()
+	{
+		create();
+		this.m.Name = "Race: Vampire";
+		this.m.IsHidden = false;
+	}
+
+	o.getTooltip = function()	// This function doesn't exist yet in the vanilla effect, only in its parent
+	{
+		return [
+			{
+				id = 1,
+				type = "title",
+				text = this.getName()
+			},
+			{
+				id = 2,
+				type = "description",
+				text = this.getDescription()
+			},
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/melee_defense.png",
+				text = "Heal " + ::MSU.Text.colorGreen("100%") + " of Hitpoint damage inflicted on enemies"
```

Comment:
Remove the `on enemies` part.

Reply:
In this case I copied the way that vanilla does the description of their necrosavant potion effect. They write:
`"Heal [color=" + this.Const.UI.Color.PositiveValue + "]25%[/color] of hitpoint damage inflicted on adjacent enemies that have blood"`

So you prefer it without the enemies clause?

Reply:
I see. In that case we can't remove the `enemies` clause because we need to specify `enemies with blood`. And if we are changing the `vampire_racial` to also only heal from enemies with blood then you should write in the tooltip `on enemies with blood`.

Reply:
It's a sensible change but I will make a new branch for that